.clj-kondo/config.edn

```diff
@@ -442,7 +442,7 @@
   metabase.api.search-test/do-test-users                                               clojure.core/let
   metabase.async.api-response-test/with-response                                       clojure.core/let
   metabase.dashboard-subscription-test/with-dashboard-sub-for-card                     clojure.core/let
-  metabase.db.custom-migrations/defmigration                                           clj-kondo.lint-as/def-catch-all
+  metabase.db.custom-migrations/define-migration                                       clj-kondo.lint-as/def-catch-all
```

Comment:
I also renamed this to make it more consistent with `define-reversible-migration`, and the convention that macros be named `define-...`

Reply:
Where does that convention come from?

Reply:
I'm not sure where it comes from, I've just seen a tendency to prefer `define-multiple-word-macro` over `defmultiple-word-macro`. `defmigration` is fine because it's one word. But if we want the two names to be consistent, instead of renaming `define-reversible-migration` to `defreversible-migration`, we should rename `defmigration` to `define-migration`.